packages/core-bridge/src/runtime.rs

```diff
@@ -147,7 +147,7 @@ pub fn start_bridge_loop(channel: Arc<Channel>, receiver: &mut UnboundedReceiver
                     // returned client directly at the moment, when we repurpose the client to be
                     // used by a Worker, `init_worker` will attach the correct metrics meter for
                     // us.
-                    tokio::spawn(async move {
+                    core_runtime.tokio_handle().spawn(async move {
```

Comment:
Can we not use `tokio::spawn` anymore? Doesn't this call reference the "current" tokio runtime?

Reply:
This is more consistent. It'd be possible after a refactor for a different tokio runtime to be the current context for the thread. The code being like this makes it clear that the same runtime that lives in `CoreRuntime` and is used for the overall loop should be used for the operations within it.

Reply:
> Can we not use tokio::spawn anymore? Doesn't this call reference the "current" tokio runtime?

Yes, we can... But I agree with Spencer that explicitly using the tokio runtime handle for this makes the code easier to understand for Tokio neophytes.